### PR TITLE
Don't require date field for HOPKINS or PAGE-MP.

### LINKED ISSUE #266: PAGE-MP and HOPKINS mediated pages should not have "planned date of use"
Kind of related to #262. 

PAGE-MP and HOPKINS are not "plan in advance to visit the library" - they are just page requests with an in-between step. They should not have a "Planned date of use" field. When they are also a hold/recall, they should have a "Cancel request after" field.

screen shot 2015-07-31 at 5 18 49 pm
screen shot 2015-07-31 at 5 22 45 pm

### PR DESCRIPTION
**Pull request by jkeck**
Closes #266

### CODE REVIEW

File: app/models/requests/mediated_page.rb [hunk 1]
```diff
@@ -42,6 +42,8 @@ def item_limit
   end
 
   def requires_needed_date?
+    return false if origin == 'HOPKINS'
+    return false if origin_location == 'PAGE-MP'
```
Comment (jkeck) [inline app/models/requests/mediated_page.rb]: I think this is another indication of a sub-type that exists somewhere (similar to https://github.com/sul-dlss/sul-requests/blob/24bcfcae4cae13fbb7b5f77f6da410cf9fc82415/config/locales/en.yml#L88-L89)  :disappointed:

Comment (cbeer) [inline app/models/requests/mediated_page.rb]: Should we add a configuration for these too? It might make the code a little more clear, if nothing else.

Comment (jkeck) [inline app/models/requests/mediated_page.rb]: I thought the same thing at first but didn't really know what that config should be.  We're kind of blacklisting a library and location here so do you think something like `date_needed_library_blacklist` and `date_needed_location_blacklist` would be appropriate?

Comment (cbeer) [inline app/models/requests/mediated_page.rb]: Ah, I didn't catch that. Yeah, ew. I guess ship it and worry about it later.

**Pull request merged**